<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreUI</class>
 <widget class="QMainWindow" name="CoreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>668</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正启体简体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - Core</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="faceRecognizerSettingsLogGroupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>人脸识别设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="signButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>点击签到</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="realTimeCaptureGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像头实时画面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="realTimeCaptureLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>741</width>
       <height>601</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>使用帮助</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>  点击签到时,系统会打开检测能否打开摄像头,请等待。
  可以多人连续依次签到，也可签到后关闭此界面。
  当不能完成签到时，请检测是否遮挡了脸庞，或周围有多人。</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>选择课程</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>21</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>微波天线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>专业外语</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通信原理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>移动通信</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>信号与系统</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
